### ISSUE TITLE
Cannot configure markdown-it

### ISSUE THREAD

**Issue by samhinshaw** [reptar/reptar#84]
Following the suggestions from the [example config file](https://github.com/reptar/reptar/blob/master/lib/config/config-example.js#L138-L146), I tried to add markdown-it configuration options listed [in their readme](https://github.com/markdown-it/markdown-it#init-with-presets-and-options):

```js
// reptar.config.js
  markdown: {
    extensions: ['markdown', 'mkdown', 'mkdn', 'mkd', 'md'],
    options: {
      preset: 'commonmark',
      highlight: true,
      typographer: true,
      langPrefix: ''
    },
  },
```

However, the config file fails to validate when any options are added:
`reptar.config.js validation error: child "markdown" fails because [child "options" fails because ["langPrefix" is not allowed, "typographer" is not allowed]]`

However, I was able to configure the options with middleware: 
```js
const configureMarkdown = (reptar) => {
  reptar.renderer._md.options.langPrefix = '';
  reptar.renderer._md.options.typographer = true;
};
```

Is this the intended behavior?

**Comment by hswolff** [comment]
You are absolutely correct this is a bug. The options should accept anything.

**Comment by hswolff** [comment]
Fixed in `v3.3.0`! https://github.com/reptar/reptar/blob/master/CHANGELOG.md#bug-fixes 

Thanks for the report!
